app/views/users/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<% provide :title, t("sign_up") %>
+<h1><%= t("sign_up") %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: signup_path do |f| %>
+      <%= f.label :name, t("name_signup") %>
```

Comment:
Tìm hiểu về `Translations for Active Record Models`
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models